### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by marijanbrvar**
Developed UI for calculator

### CODE REVIEW

File: src/components/calculator.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
+/* eslint-disable react/button-has-type */
```
Comment (henatan99) [inline src/components/calculator.jsx:1]: - [ ] For the sake of following project requirements, please make this file a `.js` file not `.jsx` as there is no difference it makes here. :relaxed:

File: src/App.js [hunk 2]
```diff
@@ -1,29 +0,0 @@
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
```
Comment (henatan99) [inline src/App.js:4]: - [ ] I think you shouldn't have deleted this `App.js` file. It is a common practice to use the `App` component as the top-level wrapper of components. Kindly revert this change and import and render the `Calculator` component in the `App` component. :pray: 
- [ ] Also, kindly refactor it to a React class component as you are required to practice React components in this milestone.

File: src/index.css [hunk 3]
```diff
@@ -1,27 +1,135 @@
...
+  box-shadow: 0 0 20px 0 #aaa;
+}
+
+.calc-display {
+  color: white;
+  background: #1c191c;
+  line-height: 130px;
+  font-size: 6em;
+  flex: 1;
+}
```
Comment (henatan99) [inline src/index.css:61]: - [ ] Your design is more or less fine and it sticks with the original design in many aspects. Yet the display field looks like it needs a little more work to match it well with the original design. Particularly, the background color and the font size. :thinking: 
image
image

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @marijanbrvar , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
- [x] All the boilerplate that got created by the CRA (texts, images, and styles) ought to have gotten deleted. I think you have some of them in your public directory. Kindly remove them. 
- [ ] You can adapt the Prerequisites Setup | Install | Usage | Run tests sections in your readme from the default readme generated when you initialize your app with `create-react-app`. These sections make the life of those who clone your repo, easier. 


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashboard. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**